src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book(Long id, String title, BigDecimal price) {
+    }
```

Comment:
you've created parameterized constructor but did not use parameters and did not init your fields with them